systemd/gnome-session-x11@pantheon.target

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=GNOME X11 Session (session: %i)
+OnFailure=gnome-session-failed.target
+OnFailureJobMode=replace
+DefaultDependencies=no
+# Start happens explicitly
+RefuseManualStart=no
+# Stop happens by starting gnome-session-shutdown.target
+#RefuseManualStop=yes
```

Comment:
Should we remove this comments before merging?

Reply:
These unit files are just direct copies of the ones from `gnome-session`, just tweaked where necessary to make it run pantheon instead and these lines come from there. It seems nicer to keep them as similar as possible to the `gnome-session` ones so we can see the diff if necessary.